### ISSUE TITLE
Create EditModuleDescriptor and EditActivityDescriptor

### ISSUE THREAD

**Issue by don-tay** [AY1920S2-CS2103T-T10-4/main#112]
Each activity might need different class
- Use EditActivityDescriptor as interface

Refer to EditPersonDescriptor for inspiration

Progress
- [x]  Implement EditModuleDescriptor, EditModuleCommandParser and corresponding module related classes, if necessary
- [x] Implement EditActivityDescriptor, EditActivityCommandParser and corresponding activity related classes, if necessary.
- [x] Implement all tests below:

- [x] EditModuleDescriptorTest
- [x] EditModuleCommandTest
- [x] EditModuleCommandParserTest

- [x] EditActivityDescriptorTest
- [x] EditActivityCommandTest
- [x] EditActivityCommandParserTest

**Comment by don-tay** [comment]
`EditActivityCommandTest` will be delayed till after #116 is done. Creating new issue #117 to implement `EditActivityCommandTest`.

**Comment by don-tay** [comment]
All remaining task ported over to #118
